config/manager/manager.yaml

```diff
@@ -101,11 +101,11 @@ spec:
                 key: MESH_NAMESPACE
                 optional: true
           - name: NIM_STATE
-            valueFrom:
-              configMapKeyRef:
-                name: odh-model-controller-parameters
-                key: nim-state
-                optional: false
+            value: $(nim-state)
```

Comment:
I think it is better to hard-code some value, rather than using this legacy `$(variable)` syntax. Just to not confuse people with variables which, IIRC, we have moved away from.